configs/opencv-ovms/models/2022/config_template.json

```diff
@@ -82,6 +82,165 @@
       "plugin_config": {"PERFORMANCE_HINT": "LATENCY"},
       "target_device": "{target_device}"},
       "latest": { "num_versions": 2 }
+    },
+    {
+      "config": {
+        "name": "yolov5",
+        "base_path": "/models/yolov5s/FP16-INT8",
+        "shape": "(1,3,416,416)",
+        "nireq": 2,
+        "batch_size": "1",
+        "plugin_config": {
+          "PERFORMANCE_HINT": "LATENCY"
+        },
+        "target_device": "{target_device}"
+      },
+      "latest": {
+          "num_versions": 2
+      }
+    },
```

Comment:
don't we have yolov5s model already?

Reply:
yes, we are reusing the existing yolov5 model, but this one needs different layout

Reply:
i don't see the layout parameter you put here; did you mean the shape?  the yolov5 should get the same shape 416 by 416 if it is yolov5s